registry/src/accounts/registry.rs

```diff
@@ -0,0 +1,157 @@
+use arrayref::{array_mut_ref, array_ref, array_refs, mut_array_refs};
+use serum_common::pack::*;
+use solana_client_gen::solana_sdk::pubkey::Pubkey;
+use std::io::Read;
+
+pub const SIZE: usize = 482;
+
+// TODO: BPF overflows if we jack this up too high. Need to use raw bytes
+//       and manually index/deserialize on demand if we want to increase the
+//       length and keep the data structure as [Pubkey; CAPABILITIES_LEN].
+pub const CAPABILITIES_LEN: usize = 10;
```

Comment:
The current number of capabilities is limited due to Solana's ability to deserialize the `Registry` data structure without overflowing. 

If we want to have many more, we either need to probably do *not* deserialize via `unpack` and instead manually pull out the raw bytes from the account's data array on demand. Alternative we could change the way we're storing the `Registry`'s capabilities mapping. Probably across multiple accounts. (I'd favor the first suggestion.)